packages/blade/src/components/Checkbox/Checkbox.tsx

```diff
@@ -165,6 +167,7 @@ const _Checkbox: React.ForwardRefRenderFunction<BladeElementRef, CheckboxProps>
   const _hasError = _validationState === 'error';
   const _isDisabled = isDisabled ?? groupProps?.isDisabled;
   const _name = name ?? groupProps?.name;
+  // eslint-disable-next-line @typescript-eslint/no-unnecessary-type-assertion
```

Comment:
Not sure why this started failing suddenly. I've ignored it instead of making it the following to avoid breaking changes and unintended behaviour changes

```diff
-  const _isChecked = isChecked ?? groupProps?.state?.isChecked(value!);
+  const _isChecked = isChecked ?? groupProps?.state?.isChecked(value);
```

Reply:
Weird, should be fine to ignore. maybe some config or package got updated and tslint started throwing this.